triaging.rst

```diff
@@ -47,6 +47,35 @@ the Python Software Foundation, always follow the `PSF Code of Conduct`_.
 
    Existing members with the "Developer" role on the `issue tracker`_ are welcome to
    self-nominate for triage team membership if they are interested.
+   
+GitHub Labels
+''''''''''''''''
+
+An important component of triaging PRs and issues involves appropriately 
+categorizing them through the usage of labels. Here are several of the
+common ones utilized by the Python triage team:
```

Comment:
I think the last sentence "Here are several ...." can be omitted.

Reply:
Agreed, that's probably unnecessary filler.

Reply:
Note: I ended up adding a header for the labels section to replace the sentence, similar to the "Responsibilities include:" used for the triage team outline.